stories/examples/Card.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Card, List } from 'react-ui-core/src'
+import { CardTheme } from '../theme'
+
+const cardContent = <List items={['One', 'Two', 'Three']} />
+
+export const DefaultCard = (
+  <Card
```

Comment:
I'd like to see us getting in the habbit of using the `Theme` / `ThemeProvider` from `react-themed`.
```
import { Theme } from 'react-themed'
import { CardTheme } from './theme'

const theme = compose({}, CardTheme)

export const DefaultCard = (
  <Theme theme={theme}>
    <Card/>
  </Theme>
)
```

It then assures we properly have the decorator classes applied correctly.